developer/engine/desktop/web.config

```
@@ -4,45 +4,17 @@
         <rewrite>
             <rules>
                 <!-- Desktop engine content hasn't changed between 10.0 -> 12.0 -->
-                <rule name="Redirect 11.0/ and 12.0/ to 10.0/">
+                <rule name="Redirect 11.0/ and 12.0/ to current-version/">
                     <match url="(11|12)\.0(.*)" />
```

```suggestion
                    <match url="(1\d).0(.*)" />
```
so /13.0/ (and above) will redirect to current_version

---

Hmm. Actually, I wonder if we should no longer have the 11.0 and 12.0 redirects, because we aren't forcing a version update for each release now with the new pattern? But happy to go with your suggestion for now.